### DISCUSSION TITLE
Which file to use?

### DISCUSSION THREAD

**Discussion by maxdrohde** [tefkah/zotero-night#96] [Q&A]
I see two `.xpi` files, which should I use?

<img width="248" alt="Screenshot 2023-01-25 at 13 46 40" src="https://user-images.githubusercontent.com/25734232/214672906-676364c9-587a-4164-ab6a-4f934b6e64fd.png">

**Comment by tefkah** [comment]
theyre the same file, the first one is just to have a permalink pointing from the readme
